File: tuts/003-s3-gettingstarted/java/src/main/java/com/example/s3/S3Wrapper.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.s3;

import software.amazon.awssdk.services.s3.S3Client;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class S3Wrapper {
    private static final Logger logger = LoggerFactory.getLogger(S3Wrapper.class);
    private final S3Client client;

    public S3Wrapper(S3Client client) {
        this.client = client;
    }

    // TODO: Add wrapper methods matching CLI tutorial actions
}

File: tuts/003-s3-gettingstarted/swift/Sources/S3Wrapper.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSS3
import Foundation

public class S3Wrapper {
    let client: S3Client

    public init(client: S3Client) {
        self.client = client
    }

    // TODO: Add async throws wrapper methods matching CLI tutorial actions
}
